src/components/Modals/EditProfileModal.vue

```diff
@@ -0,0 +1,190 @@
+            </div>
+          </div>
+          <div class="mt-10">
+            <div class="flex mb-3">
+              <div class="w-1/3 self-end">
+                First Name
+              </div>
+              <div class="w-2/3">
+                <ValidationProvider rules="required" name="First name" v-slot="{ errors }">
+                  <input
+                    class="w-full pl-1 py-2 field border-b border-gray-400"
+                    :class="errors.length > 0 ? 'border-red-600' : ''"
+                    type="text"
+                    id="admin-first-name"
+                    name="admin-first-name"
+                    v-model="user.firstName"
+                    placeholder="First Name" />
+                </ValidationProvider>
+              </div>
+            </div>
+            <div class="flex mb-3">
+              <div class="w-1/3 self-end">
+                Last Name
+              </div>
+              <div class="w-2/3">
+                <ValidationProvider rules="required" name="Last name" v-slot="{ errors }">
+                  <input
+                    class="w-full pl-1 py-2 field border-b border-gray-400"
+                    :class="errors.length > 0 ? 'border-red-600' : ''"
+                    type="text"
+                    id="admin-last-name"
+                    name="admin-last-name"
+                    v-model="user.lastName"
+                    placeholder="Last Name" />
+                </ValidationProvider>
+              </div>
+            </div>
+            <div class="flex mb-3">
+              <div class="w-1/3 self-end">
+                Email
+              </div>
+              <div class="w-2/3">
+                <ValidationProvider rules="required|email" name="Email" v-slot="{ errors }">
+                  <input
+                    class="w-full pl-1 py-2 field border-b border-gray-400 text-gray-400"
+                    :class="errors.length > 0 ? 'border-red-600' : ''"
+                    type="email"
+                    id="admin-email"
+                    name="admin-email"
+                    v-model="user.email"
+                    disabled
+                    placeholder="email@host.tld" />
+                </ValidationProvider>
+              </div>
+            </div>
+            <!-- <div class="flex mb-3">
+              <div class="w-1/3 self-end">
+                Password
+              </div>
+              <div class="w-2/3">
+                <ValidationProvider rules="required" name="Password" v-slot="{ errors }">
+                  <input
+                    class="w-full pl-1 py-2 field border-b border-gray-400"
+                    :class="errors.length > 0 ? 'border-red-600' : ''"
+                    type="password"
+                    id="admin-password"
+                    name="admin-password" />
+                </ValidationProvider>
+              </div>
+            </div> -->
+            <!-- <div class="flex mb-3">
```

Comment:
Any particular reason why this is all commented out?

Reply:
Don't want to remove it incase we need to enable editing phone numbers.